superflore/utils.py

```diff
@@ -70,44 +70,54 @@ def trim_string(string, length=80):
 
 
 def get_license(l):
-    bsd_re = '^(BSD)((.)*([1234]))?'
-    gpl_re = '^(GPL)((.)*([123]))?'
```

Comment:
Without this will it overmatch on a LGPL?

Reply:
Definitely would have... I just went ahead and swapped the order.